schemas/test/en/test_timeout.json

```diff
@@ -4,7 +4,7 @@
     "schema_version": "0.0.1",
     "data_version": "0.0.3",
     "survey_id": "0",
-    "session_timeout_in_seconds": 5,
+    "session_timeout_in_seconds": 120,
```

Comment:
Can this not be slightly above 60?.

Reply:
Yeah it can, I was not sure about the value I should use.